miden/src/cli/verify.rs

```diff
@@ -62,4 +68,30 @@ impl VerifyCmd {
 
         Ok(())
     }
+
+    fn infer_defaults(&self) -> (PathBuf, PathBuf) {
+        let proof_file = if self.proof_file.exists() {
+            self.proof_file.clone()
+        } else {
+            PathBuf::from("default_proof_file.txt")
+        };
+
+        let base_name = proof_file.file_stem().expect("Invalid proof file").to_str().unwrap();
+
+        let input_file = self.input_file.clone().unwrap_or_else(|| {
+            let mut input_path =
+                proof_file.parent().unwrap_or_else(|| Path::new(".")).to_path_buf();
+            input_path.push(format!("{}.inputs", base_name));
+            input_path
+        });
+
+        let output_file = self.output_file.clone().unwrap_or_else(|| {
+            let mut output_path =
+                proof_file.parent().unwrap_or_else(|| Path::new(".")).to_path_buf();
+            output_path.push(format!("{}.outputs", base_name));
+            output_path
+        });
```

Comment:
Here, we can probably use [PathBuf::set_extension()](https://doc.rust-lang.org/std/path/struct.PathBuf.html#method.set_extension) rather than formatting the file name manually.